docs/sql-programming-guide.md

```diff
@@ -1058,7 +1059,21 @@ the Data Sources API. The following options are supported:
       The JDBC fetch size, which determines how many rows to fetch per round trip. This can help performance on JDBC drivers which default to low fetch size (eg. Oracle with 10 rows).
     </td>
   </tr>
-  
+
+  <tr>
+    <td><code>batchsize</code></td>
```

Comment:
Same here. No need to change the document in this PR. How about minimizing the code changes and remove all these unrelated changes? Thanks!